store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -597,4 +606,17 @@ private String toTypeName(short type) {
         }
         return "Unrecognized type " + type;
     }
-}
\ No newline at end of file
+
+    /**
+     * Returns the storage type name for the given volume info type.
+     */
+    private String extractStorageType(VolumeInfo volInfo) {
+        String result = "";
+        if (null != volInfo.getVolumeExternalInfo()) {
+            result = volInfo.getVolumeExternalInfo().getStorageType();
```

Comment:
This seems to be our tech debt, we will revisit this.

Reply:
Yes, as I dint wanted to touch vol-info class managed here in volumecli class with small function